feature-libs/checkout/base/components/checkout-review/checkout-review-shipping/checkout-review-shipping.component.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component } from '@angular/core';
+import {
+  ActiveCartFacade,
+  CartOutlets,
+  DeliveryMode,
+  OrderEntry,
+} from '@spartacus/cart/base/root';
+import {
+  CheckoutDeliveryAddressFacade,
+  CheckoutDeliveryModesFacade,
+  CheckoutStep,
+  CheckoutStepType,
+} from '@spartacus/checkout/base/root';
+import { Address, TranslationService } from '@spartacus/core';
+import { Card, ICON_TYPE } from '@spartacus/storefront';
+import { combineLatest, Observable } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
+import { CheckoutStepService } from '../../services';
+
+@Component({
+  selector: 'cx-checkout-review-shipping',
+  templateUrl: './checkout-review-shipping.component.html',
+})
+export class CheckoutReviewShippingComponent {
+  readonly cartOutlets = CartOutlets;
+  iconTypes = ICON_TYPE;
+
+  checkoutStepTypeDeliveryAddress = CheckoutStepType.DELIVERY_ADDRESS;
+  checkoutStepTypeDeliveryMode = CheckoutStepType.DELIVERY_MODE;
+
+  constructor(
+    protected activeCartFacade: ActiveCartFacade,
+    protected checkoutDeliveryModesFacade: CheckoutDeliveryModesFacade,
+    protected checkoutDeliveryAddressFacade: CheckoutDeliveryAddressFacade,
+    protected translationService: TranslationService,
+    protected checkoutStepService: CheckoutStepService
+  ) {}
+
+  get entries$(): Observable<OrderEntry[]> {
+    return this.activeCartFacade.getDeliveryEntries();
+  }
+
+  steps$: Observable<CheckoutStep[]> = this.checkoutStepService.steps$;
```

Comment:
no need consider steps in this component.

Reply:
I kept the steps because its used in the getCheckoutStepUrl() function when editing a shipping address or delivery mode, let me know what you think about that

Reply:
we do not need this `steps$`, but we need this, otherwise click pencil will redirect to first step:
```
checkoutStepTypeDeliveryAddress = CheckoutStepType.DELIVERY_ADDRESS;
 checkoutStepTypeDeliveryMode = CheckoutStepType.DELIVERY_MODE;
```